apps/meteor/app/lib/server/lib/notifyUsersOnMessage.js

```diff
@@ -191,4 +191,9 @@ export async function notifyUsersOnMessage(message, room) {
 	return message;
 }
 
-callbacks.add('afterSaveMessage', (message, room) => notifyUsersOnMessage(message, room), callbacks.priority.LOW, 'notifyUsersOnMessage');
+callbacks.add(
+	'afterSaveMessage',
+	(message, room) => notifyUsersOnMessage(message, room),
+	callbacks.priority.MEDIUM,
```

Comment:
any reason to change this to MEDIUM? just curious

Reply:
This comes from the discussion feature. I changed that because I need that to run before the discussion callback, otherwise, we would end up without the correct number of messages within that room. https://github.com/RocketChat/Rocket.Chat/blob/28419b8278704677ee03a054ddfb66bff2b8f818/apps/meteor/app/lib/server/lib/notifyUsersOnMessage.js#L187